### ISSUE TITLE
Diacritic and special characters issue in generated Code

### ISSUE THREAD

**Issue by Dubbs777** [RazorGenerator/RazorGenerator#65]
I work on VS2015, and have an issue with the way special characters are handled. I did not have the issue on VS2013. 

For instance : 
```C#
   @{ 
        string test = "c'est l'été et ça c'est bien";
        <div>@test</div>
    }
```

generates the following code  :

```C#
string test = "c'est l'�t� et �a c'est bien"; // my accents and cedilla are gone ! 
WriteLiteral("        <div>");
Write(test);
WriteLiteral("</div>\r\n");
```

Is there a setup somewwhere I am missing?

Best regards, 
Sébastien

**Comment by pranavkm** [comment]
@davidebbo just made a change to the generator to default to UTF8 encoding when reading files. If it's the vsix, could you try updating to the latest and see if that resolves your issue?

**Comment by davidebbo** [comment]
Unless it's the other way around and the change affected you. Please specify the extension version that you are using. The latest is `1.10.1`.

See #63 for context.

**Comment by Dubbs777** [comment]
Hi, thank you for the quick answer. I used the 1.10.1 downloaded from VS2015 the first time I had the bug, I tried the one in the #63 but with both times the same issue... But I think I found out what's going on!

=> I have very few accents in our files so I did not notice right away that only one cshtml file had the issue. I "reverted" that file recently from Git after an accidental deletion. Somehow, the file got messed up by Git in a subtle way I think. How or why? I really don't know. 

I created a new cshtml file, copied the code and It seems fine now. the other files, pulled from Git are seem fine.

So it is OK for know, thank you again. Razor Generator is really a great tool !
